code/src/sixsq/nuvla/server/resources/session.clj

```diff
@@ -511,6 +508,10 @@ status, a 'set-cookie' header, and a 'location' header with the created
 ;; initialization: no schema for this parent resource
 ;;
 
+
+(def resource-metadata (gen-md/generate-metadata ::ns ::session/session))
```

Comment:
The call to `gen-md/generate-metadata` needs to be moved after crud `defmethod`s in order for scrud operations auto-discovery to work. It seems to be the case already for many resources, but not for all - this resource is an example. I didn't go through all resources to fix this yet - wanted to discuss the approach first.